man/conf.py

```diff
@@ -0,0 +1,31 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'pyenv'
+copyright = '2023, native-api'
+author = 'native-api'
+
+version = '2.3.17'
+release = '2.3.17'
```

Comment:
Absolutely. You can use bump2version or use a `sed` call.

Reply:
We have `pyenv version`

Reply:
We have `pyenv --version` specifically to get the current version.

Reply:
Hmn, are we talking about the same things? :thinking: 

You probably mean to change this file in a consistent and automated way. That's what [bump2version](https://github.com/c4urself/bump2version) does. In its config file you add the current version and all the files that contain this information. When you want to change the version, it does:

1. Raise the version.
2. Change all version strings in all affected files.
3. If configured, it can automatically commit the version change.

Certainly, with `pyenv --version` you get the current version. However, you need to attach this information in some way to `man/config.py`. How would you do that?

Reply:
`conf.py` is a program. Which means it can retrieve the version number itself.
In accordance with the DRY principle, we should only have the number in one place.

Reply:
> `conf.py` is a program. Which means it can retrieve the version number itself. In accordance with the DRY principle, we should only have the number in one place.

That true. It _seems_ we can call `libexec/pyenv---version`. If there is no other, better way to do that I can implement that.